hw2/05-form.html

```diff
@@ -7,13 +7,89 @@
     <title>05 - Form</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="05-form.js"></script>
   </head>
-  <body>
-    <main>
-      <form>
-        <h1>Exercise 05 - Form</h1>
+  <body class="bg-dark pt-5">
+    <main class="card mx-auto col-lg-4 col-md-5 col-sm-10 p-2">
+      <form id="inputForm">
+        <h1 class="text-center p-2">Exercise 05 - Form</h1>
+        <div class="form-group p-2">
+          <label for="fullName" class="h6"> Full Name: </label>
+          <input type="text" id="fullName" class="form-control" required />
+        </div>
+        <div class="form-group p-2">
+          <label for="email" class="h6"> Email </label>
+          <input type="text" id="email" class="form-control" required />
```

Comment:
It's better to change the input type of the email field to "email".

Reply:
Updated! I created a third [PR ](https://github.com/shrutidaundkar/frontend-homework/pull/3)and merged the new changes with the main branch. Thank you!